IOMC/RandomEngine/python/IOMC_cff.py

```diff
@@ -169,15 +169,11 @@
 
 randomEngineStateProducer = cms.EDProducer("RandomEngineStateProducer")
 
-def _modifyRandomNumberGeneratorServiceForPhase2( object ):
-    object.simMuonGEMDigis = cms.PSet(
-        initialSeed = cms.untracked.uint32(1234567),
-        engineName = cms.untracked.string('HepJamesRandom')
-    )
-    object.simMuonME0Digis = cms.PSet(
+from Configuration.StandardSequences.Eras import eras
+eras.run3_GEM.toModify(RandomNumberGeneratorService, simMuonGEMDigis = dict(
         initialSeed = cms.untracked.uint32(1234567),
-        engineName = cms.untracked.string('HepJamesRandom')
-    )
+        engineName = cms.untracked.string('HepJamesRandom')) )
```

Comment:
I didn't realize that `simMuonGEMDigis` does not already exist in `RandomNumberGeneratorService`. Becuase it does not exist you can't use a dictionary and instead must declare a new PSet
```python
eras.run3_GEM.toModify(RandomNumberGeneratorService, simMuonGEMDigis = cms.PSet(
                                         initialSeed = cms.untracked.uint32(1234567),
                                         engineName = cms.untracked.string('HepJamesRandom')) )
```